File: fix.dart

import 'dart:io';
void main() {
  final file = File('lib/services/call_card_ocr_parse_service.dart');
  var lines = file.readAsLinesSync();
  lines = lines.where((line) => !line.contains('smart_ocr_service.dart')).toList();
  lines = lines.map((line) {
    if (line.contains('return await SmartOcrService.enhance(')) {
      return '    return logData;';
    }
    return line;
  }).toList();
  file.writeAsStringSync(lines.join('\n'));
}

File: fix.py

with open('lib/services/call_card_ocr_parse_service.dart', 'r', encoding='utf-8') as f:
    c = f.read()
c = c.replace('import \'smart_ocr_service.dart\'; // 신규 추가', '')
c = c.replace('return await SmartOcrService.enhance(logData, recognizedText.text);', 'return logData;')
with open('lib/services/call_card_ocr_parse_service.dart', 'w', encoding='utf-8') as f:
    f.write(c)

